install_remove_pod/before/SampleApp.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		0AEC166B36EA4BBAB35AEB94 /* libPods.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 061523DA9D2646ACA1EF295C /* libPods.a */; };
```

Comment:
this is a big nit but this should not be there. I think we should have run `pod deintegrate` on this project to clean it up.